vertx-web-graphql/src/main/java/io/vertx/ext/web/handler/graphql/impl/ErrorUtil.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 Red Hat, Inc.
+ *
+ * Red Hat licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.vertx.ext.web.handler.graphql.impl;
+
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.common.WebEnvironment;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+public class ErrorUtil {
```

Comment:
Maybe we should externalize this to `common` and agree on a pattern?

https://github.com/vert-x3/vertx-web/blob/master/vertx-web/src/main/java/io/vertx/ext/web/handler/impl/ErrorHandlerImpl.java#L169-L182

We also do a similar encoding in `ErrorHandler`

Reply:
In this case the format is described by the GraphQL spec: https://github.com/graphql/graphql-spec/blob/main/spec/Section%207%20--%20Response.md#errors

So I'd rather stick to it.